src/scripts/eastern_kingdoms/dun_morogh/gnomeregan/gnomeregan.cpp

```diff
@@ -35,33 +35,33 @@ EndContentData */
 
 enum
 {
-    SAY_START                   = -1090000,
-    SAY_INTRO_1                 = -1090001,
-    SAY_INTRO_2                 = -1090002,
-    SAY_INTRO_3                 = -1090003,
-    SAY_INTRO_4                 = -1090004,
-    SAY_LOOK_1                  = -1090005,
-    SAY_HEAR_1                  = -1090006,
```

Comment:
Script text -1090006 has emote = 5 (Exclamation), while broadcast text id 4132 which u replace it with currently has no emote, but you do not update the broadcast text in your migration.